From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect console include in consolemap
Patch-mainline: never, kabi

Patch
 patches.fixes/vt-push-the-tty_lock-down-into-the-map-handling.patch
added an include into consolemap.c which caused that some functions
became defined.

Let's just ignore the include as this is only an include.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/tty/vt/consolemap.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/tty/vt/consolemap.c
+++ b/drivers/tty/vt/consolemap.c
@@ -19,7 +19,9 @@
 #include <linux/init.h>
 #include <linux/tty.h>
 #include <asm/uaccess.h>
+#ifndef __GENKSYMS__
 #include <linux/console.h>
+#endif
 #include <linux/consolemap.h>
 #include <linux/vt_kern.h>
 
